app/controllers/work_controller.rb

```diff
@@ -257,6 +257,42 @@ def record_deed(work, deed_type, user)
     update_search_attempt_contributions
   end
 
+  def show
+    # Set meta information for work pages for better archival
+    @page_title = "#{@work.title} - #{@collection.title}"
+    @meta_description = "Historical work: #{@work.title} by #{@work.author} in the #{@collection.title} collection. #{@work.description}".truncate(160)
```

Comment:
Change "Historical work" to "Historical document" here, since we do support things that are far from literature.